### DISCUSSION TITLE
Is it possible to automatically align pdf preview after build?

### DISCUSSION THREAD

**Discussion by edodeso** [overleaf-workshop/Overleaf-Workshop#152] [Q&A] [answered]
When I build with Ctrl+S or Ctrl+Alt+B, the pdf output goes to the first page, forcing me to scroll back down or do Ctrl+Alt+J. Is it possible to do this automatically?
Thanks!

**Comment by iamhyc** [comment, marked as answer]
The pdf viewer should always try to keep with the previous position. It should be a bug.

Please file a bug issue and let me know how to reproduce the situation. Thank you!

**Reply by edodeso** [reply]
I really don't know what happened neither how to reproduce that, BUT I just created a new project and copy-pasted the entire content of the one that was not working correctly and now everything seems to be ok. I'll let you know if it happens again.

For now, thanks for the quick response and for your amazing work!

**Reply by NixonZ** [reply]
Hi @iamhyc I am facing the same issue, not sure how to reproduce the issue as it works correctly when I create a new project and copy-paste the entire content.

In my case I am working on a shared overleaf project with my collaborators so creating a new project is not a solution.

I have tried uninstalling the extension, clearing the extension cache and reinstalling but that did not work.

Btw, thanks for the amazing extension!
